pipeline/pipeline.go

```diff
@@ -728,6 +728,9 @@ func (p *Pipeline) EnableEventLog() {
 }
 
 func (p *Pipeline) GetEventLogItem(index int) string {
```

Comment:
Its functions for tests. Move it to another file, with build tag? Or inline this logic to tests

Reply:
maybe just add a comment ?